docs/install_guides/installing-openvino-yum.md

```diff
@@ -14,6 +14,12 @@ The complete list of supported hardware is available in the [Release Notes](http
 
 - Red Hat Enterprise Linux 8, 64-bit
 
+Software
+
+- `CMake 3.13 or higher, 64-bit <https://cmake.org/download/>`_
+- GCC* 8.2.0
+- `Python 3.6 - 3.9, 64-bit <https://www.python.org/downloads/windows/>`_
+
```

Comment:
Please apply the same changes here. This is not included in Sphinx directives so we the hyperlinks need use Markdown markup. Or we can also use Sphinx directives to include this in a tab. 
Asterisks are not necessary for our doc site.

Reply:
Fixed